design/0001-metallb-crd.md

```diff
@@ -0,0 +1,173 @@
+# MetalLB Custom Resource Controller 
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+- [Testing](#testing)
+- [Debugging and Troubleshooting](#debugging-and-troubleshooting)
+<!-- /toc -->
```

Comment:
I didn't include the TOC in my enhancement.

This is a question for the maintainers. We should probably decide on a template rather than loosely basing on KE template otherwise every enhancement will look different.

Reply:
ha, I'm OK with including a template.  I'm also just happy to have people working on this type of doc, even if they all look different.  :)